### ISSUE TITLE
is there source code of your demo web controlled fan?

### ISSUE THREAD

**Issue by youxiaojie** [neu-rah/ArduinoMenu#349]
is there source code of your demo web controlled fan?

**Comment by neu-rah** [comment]
yes, https://github.com/neu-rah/ArduinoMenu/tree/master/examples/esp8266_EscControl
its a brushless ESC motor controled over web...
